lib/screens/generic_item_selector_screen.dart

```diff
@@ -0,0 +1,165 @@
+  List<IconData> _icons = [
+    FontAwesomeIcons.car,
+    FontAwesomeIcons.bed,
+    FontAwesomeIcons.walking,
+    FontAwesomeIcons.biking,
+  ];
+
+
+  Widget _buildIcon(int index) {
+    return GestureDetector(
+      onTap: () {
+        setState(() {
+          _selectedIndex = index;
+        });
+      },
+      child: Container(
+        height: 60.0,
+        width: 60.0,
+        decoration: BoxDecoration(
+          color: _selectedIndex == index
+              ? Theme.of(context).accentColor
+              : Color(0xFFE7EBEE),
+          borderRadius: BorderRadius.circular(30.0),
+        ),
+        child: Icon(
+          _icons[index],
+          size: 25.0,
+          color: _selectedIndex == index
+              ? Theme.of(context).primaryColor
+              : Color(0xFFB4C1C4),
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+
+    const mockResults = <AppProfile>[
+      AppProfile('John Doe', 'jdoe@flutter.io',
+          'https://d2gg9evh47fn9z.cloudfront.net/800px_COLOURBOX4057996.jpg'),
+      AppProfile('Paul', 'paul@google.com',
+          'https://mbtskoudsalg.com/images/person-stock-image-png.png'),
+      AppProfile('Fred', 'fred@google.com',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Brian', 'brian@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('John', 'john@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Thomas', 'thomas@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Nelly', 'nelly@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Marie', 'marie@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Charlie', 'charlie@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Diana', 'diana@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Ernie', 'ernie@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+      AppProfile('Gina', 'fred@flutter.io',
+          'https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png'),
+    ];
+
+    return MaterialApp(
+      title: "Hello",
+      home: Scaffold(
+        appBar: AppBar(title: const Text("AppBarTitle")),
+        body: ListView(
+          padding: EdgeInsets.symmetric(vertical: 30.0),
+          children: <Widget>[
+            ChipsInput(
```

Comment:
While we still don't have models, we now have a chip for texts over here.

TODO:
- [ ] Improve layout
- [ ] Get filter from parent acitivity if available
- [ ] Use actual models

![image](https://user-images.githubusercontent.com/21351900/149644988-2dcc34d9-3af3-40a6-9b98-7b729271b28e.png)

![image](https://user-images.githubusercontent.com/21351900/149644985-9bc50861-482b-4e36-96a1-6fb5e5495607.png)